backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs

```diff
@@ -20,8 +20,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.ReturnDate, src => src.ReturnDate)
                 .Map(dest => dest.ChequeNumber, src => src.ChequeNumber)
                 .Map(dest => dest.PayeeName, src => src.PayeeName)
-                .Map(dest => dest.PayeeAddress, src => src.PayeeAddress)
-                .Map(dest => dest.TerminationNote, src => src.TerminationNote);
```

Comment:
@asanchezr same here.

Reply:
* Payee Address should remain in the model/DB - not used in the UI (yet) but DB/model should still support it
* Termination Note was removed from row items and moved into parent Lease entity as TerminationNote I believe - can you confirm this?

Reply:
If we need payee address we'll need to get Doug to add it back in in a PSP_16_01 release. @dfilteau can you provide some background on why this was removed?
confirmed, termination note was moved to the lease.

Reply:
I found it. It is called RETURN_NOTES on PIMS_LEASE table. TerminationNotes can be removed safely. Please keep Payee Address

Reply:
@devinleighsmith I still see the field in the DB schema png
![image](https://user-images.githubusercontent.com/24322224/143965640-8c58e920-b44e-45f5-9297-acd3b04a6dd6.png)

Reply:
I would assume Doug's tool should not remove it....

Reply:
Oh, no nvm, I'm being thick, its clearly still there sorry.